index.js

```diff
@@ -33,75 +35,75 @@ Redis.prototype.connect = function () {
     db: this.db
   });
 
-  let _this = this;
-  this.client.on('ready', function () {
+  const _this = this;
+  this.client.on('ready', () => {
     R5.out.log(`Connected to Redis (db: ${this.db})`);
     _this.ready = true;
   });
 
-  this.client.on('error', function (err) {
+  this.client.on('error', err => {
     R5.out.error(`Redis error: ${err}`);
     _this.ready = false;
     _this.connect();
   });
 };
 
-Redis.prototype.get = function (key, callback) {
-  if (!callback) { callback = function () { }; }
-
-  this.client.get(key, function (err, data) {
-    if (err) { R5.out.error(err); }
+Redis.prototype.handle_client_oper_action = function (action, key, callback) {
+  does_callback_exist(callback);
+  this.client[action](key, (err, data) => {
+    handle_err_log(err);
     callback(err, data);
   });
 };
 
+Redis.prototype.get = function (key, callback) {
+  this.handle_client_oper_action('get', key, callback);
+};
+
 Redis.prototype.set = function (key, value, key_expiration, callback) {
-  if (!callback) { callback = function () { }; }
+  does_callback_exist(callback);
 
   if (typeof value === 'object') {
-    R5.out.log('Passing a string into set is recommended (currently passed in object)');
+    R5.out.log(
+      'Passing a string into set is recommended (currently passed in object)'
+    );
     value = stringify(value);
   }
 
   if (key_expiration) {
     if (value) {
       this.client.set(key, value, 'EX', key_expiration, handle_data);
-    }
-    else {
+    } else {
```

Comment:
I prefer ifs/elsifs/elses on separate lines from the brackets:
```
if () {
  ...
}
else if () {
  ...
}
else {
  ...
}
```

Reply:
So that we should be using brace-style stroustrup, Correct?